addons/calendar/models/calendar_attendee.py

```diff
@@ -148,14 +148,16 @@ def do_accept(self):
         """ Marks event invitation as Accepted. """
         for attendee in self:
             attendee.event_id.message_post(
-                body=_("%s has accepted invitation") % (attendee.common_name),
-                subtype_xmlid="calendar.subtype_invitation")
+                body=_("%s has accepted the invitation") % (attendee.common_name),
+                subtype_xmlid="calendar.subtype_invitation",
+                author_id=attendee.partner_id.id)
```

Comment:
While ranging :socks:  could you order alphabetically ? + add a final virgule + parenthèse at next line, like
```
    attendee.event_id.message_post(
        author_id=attendee.partner_id.id,
        body=_("%s has accepted the invitation") % (attendee.common_name),
        subtype_xmlid="calendar.subtype_invitation",
    )
```